edualgo/maths/Fibonacci.py

```diff
@@ -0,0 +1,13 @@
+import math
+
```

Comment:
Add the solution of the problem in the following methods as well,
- Dynamic Programming